README.md

```diff
@@ -184,7 +184,7 @@ import React from 'react'
 
 export default class Error extends React.Component {
   static getInitialProps ({ res, xhr }) {
-    const statusCode = res ? res.statusCode : xhr.status
+    const statusCode = res ? res.statusCode : (xhr ? xhr.status : 500)
```

Comment:
We can add a note to handle `null` in the error component? "Runtime error" type thing